npm/README.md

```diff
@@ -0,0 +1,38 @@
+# Bazel watcher
```

Comment:
can you avoid checking in a copy of the file? just change publish.sh to get the README from / instead?

Reply:
Rm'd